instrumentation/othttp/handler.go

```diff
@@ -120,6 +120,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	h.handler.ServeHTTP(rww, r.WithContext(ctx))
 
 	setAfterServeAttributes(span, bw.read, rww.written, rww.statusCode, bw.err, rww.err)
+	span.SetStatus(standard.SpanStatusFromHTTPStatusCode(int(rww.statusCode)))
```

Comment:
```suggestion
	span.SetStatus(standard.SpanStatusFromHTTPStatusCode(rww.statusCode))
```

The cast to `int` isn't necessary as the `rww.statusCode` field is already an `int`.

Reply:
I thought about that too. But as it's setting attributes I changed it be outside `setAfterServeAttributes`, I guess it makes sense. If needed to be within `setAfterServeAttributes` I can change it.